Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
I opened bug 8463791 for this.

But I don't think we should put in the workaround that you have now for WeekOfMonth in Year. It's insufficient and not the right way to address the problem.

---
In reply to: [74468177](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74468177) [](ancestors = 74468177,74461625,74359393,74356593,74354002,74342094,74336418)